libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2839,6 +2846,17 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 			);
 
 		if (
+			funType->kind() == FunctionType::Kind::ArrayPop &&
+			exprType->nameable() &&
```

Comment:
I know that `nestedMapping` has an assert for `nameable`. Wondering if we need the `exprType->nameable()` condition here? Do you have an example that throws otherwise?

Reply:
You're right it is not needed at this particular location. Only at the other place where I added it. There it would throw in `signed_rational_modulus.sol`. Will remove it here.